modules/react-components/src/components/tab/resource-tab/resource-tab.tsx

```diff
@@ -70,6 +75,16 @@ export const ResourceTab: FunctionComponent<ResourceTabPropsInterface> & Resourc
         setActiveIndex(defaultActiveIndex);
     }, [ defaultActiveIndex ]);
 
+    /**
+     * Called to set the panes list length initially.
+     */
+    useEffect(()=> {
```

Comment:
This will be called multiple times. If you just want this to be called one time, set `[]` as the dependency array.